Sources/App/Core/Github.swift

```diff
@@ -417,6 +370,25 @@ extension Github {
             }
         }
 
+        struct FundingLinkNode: Codable, Equatable {
+            enum Platform: String, Codable {
+                case communityBridge = "COMMUNITYBRIDGE"
+                case customUrl = "CUSTOMURL"
+                case gitHub = "GITHUB"
+                case issueHunt = "ISSUEHUNT"
+                case koFi = "KOFI"
+                case lfxCrowdfunding = "LFXCROWDFUNDING"
+                case liberaPay = "LIBERAPAY"
+                case openCollective = "OPENCOLLECTIVE"
+                case otechie = "OTECHIE"
+                case patreon = "PATREON"
+                case tideLift = "TIDELIFT"
+            }
+
+            var platform: Platform
+            var url: String
+        }
```

Comment:
This "copy" of the type is the one that carries the GH specific parts. Note that I've simplified the decode based on the assumption (from what I've seen in the GraphQL response) that the platform strings are always uppercase.

If they're not, this would need adjusting.

Reply:
Unfortunately the underscores in the GH responses are really inconsistent and they need hard coding as in my original version. I'll update this.

Reply:
Ah, my bad, I’d copied over the camel case names and used the uppercase transformation. That nuked the underscores.